feat: specify custom ca key (#114)
kubeshop/testkube-docs: docs/articles/private-ca.md

kubectl -n <namespace> create secret generic <secret name> --from-file=ca.crt=<path to the file with the ca bundle>
```

:::tip

> If your CA bundle secret is generated by some external system and you do cannot

Reviewer: ```suggestion
If your CA bundle secret is generated by an external system and you cannot
```

However, if the containers specified as part of the workflow require the trust
of your private CA then you will need to configure the container/pod in a
similar way as shown in the global template to mount the CA bundle to the right

> location and/or if necessary specify an environment variable.

Reviewer: ```suggestion
location and/or, if necessary, specify an environment variable.
```
